src/components/HomePage.js

```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+
+function HomePage () {
+
+        return (
+            <div>
+            <h1>This is the Home Page</h1>
+            <section>
+                <img src= "./assets/beers.png" alt="Beers"/>
+                <h2><Link to="/beers">Beers</Link></h2>
+            </section>
+    
+            <section>
+                <img src="./assets/random-beer.png" alt="Random Beers"/>
+                <h2><Link to="/random-beers">Random Beers</Link></h2>
```

Comment:
Here you have a mismatch between your `Link` and the `Route` (the same for the new-beer)

```suggestion
                <h2><Link to="/random-beer">Random Beers</Link></h2>
```